### DISCUSSION TITLE
Where should I choose 1-wire operation for DHT22 (AM2302)?

### DISCUSSION THREAD

**Discussion by cleverpal-ai** [arendst/Tasmota#16132] [Q&A] [answered]
I've got a couple of Sonoffs (Mini / Mini R2) powered by Tasmota and my objective is to connect 2-3 humidity sensors to each device. AM2302 alone works pretty fine, but as soon as I connect another AM2302 sensor in parallel (like in Tasmota official docs mentioned for temperature/humidity sensors) 1-wire, I see "null" figures for my original sensor and no second sensor appearing in the list. What should I do for using the parallel connection of humidity sensors, anything to change in the firmware source? 

Thanks!

**Comment by barbudor** [comment, marked as answer]
Tasmota is supposed to support up to 4 of those sensors in parallel on the same GPIO

However it may depends:
- On the quality of the sensor : originals are already not very good, clones are worst
- Length of the cables (over a few tenth of centimeters, do not expect them to work properly)
- Proper cabling : single wire with all sensors on the wire is generally much better than star wiring

Consider using better sensors such as BME280 and keep wires shorts

**Reply by cleverpal-ai** [reply]
I have this kind of wiring of 2 sensors (both tested individually & working fine)
![image](https://user-images.githubusercontent.com/62514634/182234320-5faa744a-6a2c-4c58-ae6d-f366ccbb71e2.png)
Should I expect 2nd sensor to appear in Tasmota's web interface after I connect it?
The only thing I've changed in a custom-compiled FW is - adding DHT support (tasmota-lite).

**Reply by cleverpal-ai** [reply]
As far as I understand Humidity sensors are not capable of operating while being connected using 1-wire scheme on Sonoff Mini.
It seems that I need to use a separate GPIO. The box of Sonoff is too tight, thus I planned to save some wiring space on 1-wire.
